docker/puppetserver-standalone/docker-entrypoint.d/90-consul.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+if [ "$CONSUL_ENABLED" = "true" ]; then
+  ipaddress="$(ifconfig eth0 | grep 'inet addr' | cut -d ':' -f 2 | cut -d ' ' -f 1)"
+  consul_hostname="${CONSUL_HOSTNAME:-consul}"
+  consul_port="${CONSUL_PORT:-8500}"
+
+  cat <<SERVICEDEF > /puppet-service.json
+{
+  "name": "puppet",
+  "id": "$HOSTNAME",
+  "port": 8140,
+  "address": "$ipaddress",
+  "checks": [
+    {
+      "http": "https://$HOSTNAME:8140/production/status/test",
```

Comment:
Should we use the ENV variable for the environment here?

This one: https://github.com/puppetlabs/puppetserver/blob/master/docker/puppetserver-standalone/Dockerfile#L18

Reply:
woops I made that change locally and forgot to push it :P